src/rules/functionality.ts

```diff
@@ -93,6 +101,8 @@ export default {
   'triple-equals': [true, 'allow-null-check'],
   // Makes sure result of `typeof` is compared to correct string values
   'typeof-compare': true,
+  // Warns if an explicitly specified type argument is the default for that type parameter. <FEEDBACK>
+  'use-default-type-parameter': false,
```

Comment:
[Examples](https://github.com/palantir/tslint/blob/master/test/rules/use-default-type-parameter/test.ts.lint).

Doesn't flag any neutron errors.  I guess there's no harm in turning it on?